spec/draft-barnes-cfrg-4q.md

```diff
@@ -662,26 +665,22 @@ DH(m, P):
 Return [m]*Q in affine coordinates
 ~~~~
 
+The role of the separate multiplication by 392 is to ensure that Q is an N-torsion
+point so that the scalar multiplication algorithms above may be used safely to produce
+correct results.  In other words, as the cofactor is greater than one, Curve4Q MUST NOT be used with
+ordinary Diffie-Hellman; it MUST always be used for Diffie-Hellman
+with cofactor.  Note that the multiplication by the cofactor 392 can be safely
```

Comment:
To a lay-ish person, "MUST NOT be used with ordinary DH" sounds very restrictive, when really you just mean you have to do cofactor clearing.  Rephrase as "MUST do cofactor clearing" rather than "MUST NOT do normal".